packages/api/tests/endpoints/test-granules.js

```diff
@@ -2937,86 +2695,12 @@ test.serial('PATCH() does not write to DynamoDB/Elasticsearch/SNS if writing to
   await patch(expressRequest, response);
   t.true(response.boom.badRequest.calledWithMatch('something bad'));
 
-  const actualPgGranule = await t.context.granulePgModel.get(t.context.knex, {
-    cumulus_id: newPgGranule.cumulus_id,
-  });
-
-  const actualApiGranule = await translatePostgresGranuleToApiGranule({
-    granulePgRecord: actualPgGranule,
-    knexOrTransaction: knex,
-  });
-
   t.deepEqual(
     await t.context.granulePgModel.get(t.context.knex, {
       cumulus_id: newPgGranule.cumulus_id,
     }),
     newPgGranule
   );
-  t.deepEqual(await t.context.esGranulesClient.get(actualApiGranule.granuleId), esRecord);
-
-  const { Messages } = await sqs()
-    .receiveMessage({
-      QueueUrl: t.context.QueueUrl,
-      WaitTimeSeconds: 10,
-    });
-
-  t.is(Messages.length, 0);
-});
-
-test.serial('PATCH rolls back PostgreSQL records and does not write to SNS if writing to Elasticsearch fails', async (t) => {
```

Comment:
Can we make it a test to:
PATCH does not write to SNS if writing to PostgreSQL fails

Reply:
I think it's a good idea.